concent_api/core/payments/base.py

```diff
@@ -64,3 +64,8 @@ def is_provider_account_status_positive(backend, request = None):
 @_add_backend
 def calculate_amount_pending(backend):
     return backend.calculate_amount_pending()
+
+
+@_add_backend
+def is_requestor_account_status_positive(backend, request):
```

Comment:
This function will be added in https://github.com/golemfactory/concent/pull/213 but called `is_provider_account_status_positive`. But actually it uses `requestor_eth_address` :) We should make it independent, so please just call it `is_account_status_positive`, I will add comment to the @kbeker PR.

Reply:
ok, I have added `TODO` not to forget to change it once `concent_rpc` becomes available